themes/x-mas/template.html

```diff
@@ -0,0 +1,2222 @@
+<style>
+  :host {
+    --media-primary-color: black;
+    --media-tertiary-color: var(--media-accent-color, #7596cc);
+    --media-text-color: white;
+    --media-control-background: transparent;
+    --media-control-hover-background: transparent;
+
+    --media-range-track-height: 6px;
+    --media-range-bar-color: repeating-linear-gradient(45deg, #e72d33, #e72d33 6px, #ffffff 6px, #ffffff 12px);
+    --media-range-track-background: rgba(255, 255, 255, 0.4);
+    --media-range-track-border-radius: 9999px;
+
+    --media-range-thumb-background: var(--media-tertiary-color);
+    --media-range-thumb-width: 14px;
+    --media-range-thumb-height: 14px;
+    --media-tooltip-display: none;
+  }
+
+  :host([mediastreamtype='live']) media-time-range,
+  :host([mediastreamtype='live']) media-time-display {
+    opacity: 0;
+  }
+
+  media-control-bar {
+    position: relative;
+    margin: 0.5rem;
+    padding: 0.5rem;
+    align-items: center;
+    display: flex;
+    gap: 0.5rem;
+  }
+
+  .icon-button {
+    position: relative;
+    flex: none;
+    display: flex;
+    align-items: center;
+    justify-items: center;
+    height: 2rem;
+    width: 2rem;
+    transition: ease-in-out 0.2s;
+    padding: 0;
+
+    * {
+      transition: ease-in-out 0.2s all;
+    }
+
+    &:hover {
+      svg {
+        transition: ease in-out 0.3s;
+        transform: scale(1.05);
+
+        .ornament {
+          transform: scale(1.2) translate(-4px, 0px);
+        }
+      }
+    }
+
+    svg {
+      position: absolute;
+      overflow: hidden;
+      width: 100%;
+      height: 1.25rem;
+      margin: 0 !important;
+      padding: 0 !important;
+    }
+  }
+
+  div[slot='centered-chrome'] {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: start;
+    justify-content: center;
+    overflow: hidden;
+
+    media-play-button {
+      flex: none;
+      display: flex;
+      margin-bottom: 10px;
+      align-items: center;
+      justify-items: center;
+      height: 70%;
+      width: 20%;
+      margin-top: -1rem;
+      border-radius: 9999px;
+
+      &:hover {
+      }
+
+      svg {
+        height: 100%;
+        filter: drop-shadow(0px 0px 36px #00000069);
+      }
```

Comment:
just noting CSS nesting is relatively new https://caniuse.com/?search=css%20nesting

Reply:
Fixed

Reply:
thank you! ❤️